go.mod

```diff
@@ -329,7 +328,12 @@ require (
 	k8s.io/apiextensions-apiserver v0.30.3
 	k8s.io/cli-runtime v0.30.3
 	k8s.io/kubectl v0.30.3
+	k8s.io/perf-tests/network/benchmarks/netperf v0.0.0-00010101000000-000000000000
 	sigs.k8s.io/controller-runtime v0.18.5
 )
 
+require github.com/rogpeppe/go-internal v1.12.0 // indirect
+
 replace github.com/vishvananda/netns => github.com/inspektor-gadget/netns v0.0.5-0.20230524185006-155d84c555d6
+
+replace k8s.io/perf-tests/network/benchmarks/netperf => github.com/ritwikranjan/perf-tests/network/benchmarks/netperf v0.0.0-20241001122546-074cd3ab166f
```

Comment:
commenting here to remember reverting this once the changes are upstream

Reply:
We are going to use Azure/perf-tests for now, I have opened an issue on the upstream kubernetes/perf-tests, not sure if we will be moving there or not though!

Reply:
in that case don't we want to reference `Azure/perf-test` directly?

Reply:
Please replace with https://github.com/Azure/perf-tests .